yarn.lock

```diff
@@ -8965,10 +8965,17 @@ tar@^4:
     safe-buffer "^5.1.2"
     yallist "^3.0.2"
 
-tas-client@^0.0.950:
-  version "0.0.950"
-  resolved "https://registry.yarnpkg.com/tas-client/-/tas-client-0.0.950.tgz#0fadc684721d5bc6d6af03b09e1ff5a83a5186fc"
-  integrity sha512-AvCNjvfouxJyKln+TsobOBO5KmXklL9+FlxrEPlIgaixy1TxCC2v2Vs/MflCiyHlGl+BeIStP4oAVPqo5c0pIA==
+tas-client-umd@^0.1.0:
+  version "0.1.0"
+  resolved "https://registry.yarnpkg.com/tas-client-umd/-/tas-client-umd-0.1.0.tgz#267ab9a53b7bc138068db58ea240b63a4f02af2e"
+  integrity sha512-RdD5IchlVurbMHiRUfNQgAsOvqA4iktQcWRpL+reAQv0Q9v6GliuDhUgi7v2A8LTGX4bCTQNG7lX2DqX61JuLw==
+  dependencies:
+    tas-client "^0.1.8"
+
+tas-client@^0.1.8:
```

Comment:
I would not expect this dependency to remain unless some other module depends on it.

Reply:
This, and all other comments related to the extra dependency are due to the fact that the umd version uses the regular module to mirror typings. It is functionally a dev dependency.